src/App.js

```diff
@@ -1,13 +1,66 @@
-import React from 'react';
+import React, { Component } from 'react';
```

Comment:
Good job importing `{ Component }` up here instead of dot.syntaxing line 6. I prefer doing it this way, so I can see all of my imports in one place.